cmd/types/types.go

```diff
@@ -6,3 +6,12 @@ type UserInfo struct {
 	Location    string `json:"location"`
 	PublicRepos int    `json:"public_repos"`
 }
+
+//FollowingUser contains following user information
+type FollowingUser struct {
+	Name string `json:"login"`
+	URL  string `json:"html_url"`
+}
+
+//FollowingUserList stores a list of FollowingUser
+type FollowingUserList = []FollowingUser
```

Comment:
Renamed `FollowingUserList ` to  `FollowingUsers`

Reply:
Done